cpu/atmega_common/periph/gpio.c

```diff
@@ -195,8 +281,70 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
         return -1;
     }
 
-    /* not a valid interrupt pin */
+    /* not a valid interrupt pin. Set as pcint instead if pcints are enabled */
     if (int_num < 0) {
+	    /* If pin change interrupts are enabled, enable mask and interrupt */
+        #ifdef PCINT_NUM_BANKS
+        int8_t offset = -1;
+        //uint8_t port_num = _port_num(pin);
+        uint8_t pin_num = _pin_num(pin);
+        uint8_t b;
+        for(b=0; b<PCINT_NUM_BANKS; b++) {
```

Comment:
Missing space between `for` and `(`. Same in multiple other lines. (I'd suggest to run `uncrustify`)